### PR TITLE
Add support for external_id and fail_mode. Updated syntax to pass pep…

### PR DESCRIPTION
**Pull request by markleehamilton**
…8 v1.6.2 and pylint v0.25.0

### CODE REVIEW

File: network/openvswitch_bridge.py [hunk 1]
```diff
@@ -43,62 +47,98 @@
         default: 5
         description:
             - How long to wait for ovs-vswitchd to respond
+    external_id:
+        required: false
+        description:
+            - bridge external-id
```
Comment (bcoca) [inline network/openvswitch_bridge.py]: missing version_added and default, also for fail_mode below

File: network/openvswitch_bridge.py [hunk 2]
```diff
@@ -108,27 +148,93 @@ def run(self):
...
+            'fail_mode': {'default': ''},
+            'syslogging': {'required': False, 'type': 'bool', 'default': True}
         },
         supports_check_mode=True,
     )
 
-    br = OVSBridge(module)
+    if (module.params["syslogging"]):
+        syslog.openlog('ansible-%s' % os.path.basename(__file__))
+
```
Comment (bcoca) [inline network/openvswitch_bridge.py]: module already log to syslog, no need to add it again

**Comment by markleehamilton** [comment]
I believe I've addressed everything, bcoca.

File: network/openvswitch_bridge.py [hunk 3]
```diff
@@ -43,92 +47,227 @@
...
 - openvswitch_bridge: bridge=br-int state=present
+
+# Create an integration bridge
+- openvswitch_bridge: bridge=br-int state=present fail_mode=secure
+  args:
+    external_ids:
+        bridge-id: "br-int"
 '''
 
+import syslog
```
Comment (bcoca) [inline network/openvswitch_bridge.py]: no need for syslog, modules already log to it

**Comment by markleehamilton** [comment]
syslog import has been removed

**Comment by markleehamilton** [comment]
bcoca, I've all of your comments.

**Pull request merged**